xiblint/rules/__init__.py

```diff
@@ -4,6 +4,11 @@
 from importlib import import_module
 
 
+class Rule(object):
```

Comment:
Maybe also https://docs.python.org/2/library/abc.html and an abstract `check` method?

Reply:
Not sure what's better here, I just pushed a real use of the initializer here, which might actually be preferred since likely no rule would need to override this now

Reply:
I agree. I basically only wanted a base `check` method that's not implemented.